### DISCUSSION TITLE
How not to invert the colour of the second book in Parallel View?

### DISCUSSION THREAD

**Discussion by HubKing** [crosswire/xiphos#1109] [Q&A] [answered]
As you see below, the French version's colours are inverted. How to prevent this and display both books in the same colours?

![image](https://user-images.githubusercontent.com/10900489/164456865-7ec8ff0f-1720-4543-bd1e-f0a11b24c3f4.png)

**Comment by karlkleinpaste** [comment, marked as answer]
![parallel-color-alternation](https://user-images.githubusercontent.com/7548920/164457778-9abe846c-a560-4056-8b9b-29f7b290bdc5.png)

**Reply by HubKing** [reply]
Thanks.
